opencti-platform/opencti-front/src/private/components/settings/users/UserConfidenceLevelField.tsx

```diff
@@ -128,6 +130,7 @@ const UserConfidenceLevelField: FunctionComponent<UserConfidenceLevelFieldProps>
         editContext={editContext}
         disabled={!switchValue || disabled}
         variant="edit"
+        maxLimit={effectiveConfidenceLevel?.max_confidence ?? 0}
```

Comment:
```suggestion
```

An admin shall not be limited to his own level. This is an administration feature...